gdx/src/com/badlogic/gdx/maps/tiled/BaseTmxMapLoader.java

```diff
@@ -193,12 +192,24 @@ protected void loadObject (MapLayer layer, Element element) {
 					Polyline polyline = new Polyline(vertices);
 					polyline.setPosition(x, y);
 					object = new PolylineMapObject(polyline);
-				} else if ((child = element.getChildByName("ellipse")) != null) {
-					object = new EllipseMapObject(x, y - height, width, height);
+				} else if ((child = element.getChildByName("ellipse")) != null) {                                 
+					object = new EllipseMapObject(x +width/2, y - height/2, width, height);
 				}
 			}
 			if (object == null) {
-				object = new RectangleMapObject(x, y - height, width, height);
+                            int gid = -1;
+                            if((gid = element.getIntAttribute("gid", -1)) != -1){
+                                TiledMapTile tile= map.getTileSets().getTile(gid);
```

Comment:
I just tested that, an yes, you can use animated tiles. The tmx format still gives these tiles a gid, meaning it can be retrieved from the map. You can get a TextureRegion from the returned Tile, but it will only be the current frame.

Maybe the TextureMapObject should contain the Tile, rather than only the Texture(Region)?